js/chirp.js

```diff
@@ -1,7 +1,16 @@
-function chirp(n){
+function chirp(n) {
   // FIX ME
+  let chirpStr = "chirp ";
+  if (n === 0) {
+    return;
+  }
+  if (n === 1) {
```

Comment:
The if statement here is not needed. Try to think about what would happen if you only had just the return line from the else portion of this if/else.